x/evidence/types/genesis_test.go

```diff
@@ -32,11 +32,11 @@ func TestNewGenesisState(t *testing.T) {
 		expPass  bool
 	}{
 		{
-			"cannot proto marshal",
+			"can proto marshal",
 			func() {
 				evidence = []exported.Evidence{&TestEvidence{}}
 			},
-			false,
+			true,
```

Comment:
what that a bug before?

Reply:
There was no bug. I've added `proto.Message` to  `exported.Evidence` . So now we can serialize it.